.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
```

Comment:
Dropping Node 16 testing [cuz it’ll be EOL in 3 months](https://nodejs.org/en/blog/announcements/nodejs16-eol)